src/js/samples/components/SampleRights.tsx

```diff
@@ -63,49 +78,36 @@ export const SampleRights = props => {
 
                 <InputGroup>
                     <InputLabel>Group Rights</InputLabel>
-                    <InputSelect value={props.group} onChange={e => props.onChangeRights("group", e.target.value)}>
+                    <InputSelect
+                        aria-label="group"
+                        value={group}
+                        onChange={e =>
+                            mutation.mutate({
+                                sample_group_read: includes(e.target.value, "r"),
+                                sample_group_write: includes(e.target.value, "w"),
+                            })
+                        }
+                    >
                         {options}
                     </InputSelect>
                 </InputGroup>
 
                 <InputGroup>
                     <InputLabel>All Users' Rights</InputLabel>
                     <InputSelect
-                        name="all"
-                        value={props.all}
-                        onChange={e => props.onChangeRights("all", e.target.value)}
+                        aria-label="all"
```

Comment:
Ideally you would get the label based on the content of the `InputLabel` above. Failing that, the `aria-label` should make sense for anyone using accessibility software